AsyncDisplayKit/Details/ASCollectionDataController.mm

```diff
@@ -71,7 +71,6 @@ - (void)willReloadData
     [self batchLayoutNodesFromContexts:contexts ofKind:kind completion:^(NSArray<ASCellNode *> *nodes, NSArray<NSIndexPath *> *indexPaths) {
       [self insertNodes:nodes ofKind:kind atIndexPaths:indexPaths completion:nil];
     }];
-    [_pendingContexts removeObjectForKey:kind];
```

Comment:
This change is due to the fact that `_pendingContexts` is being enumerated in this block so mutating it has no effect. What change are you asking for here?